munch/__init__.py

```diff
@@ -185,6 +185,20 @@ def __repr__(self):
     def __dir__(self):
         return list(iterkeys(self))
 
+    def __getstate__(self):
+        """ Implement a serializable interface used for pickling.
+
+        See https://docs.python.org/3.6/library/pickle.html.
+        """
+        return self.toDict()
+
+    def __setstate__(self, state):
+        """ Implement a serializable interface used for pickling.
+
+        See https://docs.python.org/3.6/library/pickle.html.
+        """
+        self.__init__(state)
```

Comment:
It's better to call `clear` and `update` here -- the behavior of calling `__init__` twice on a dict is not well defined...